diff --git a/src/plugin-interface/QvPluginBase.hpp b/src/plugin-interface/QvPluginBase.hpp
index 115ca75..4800252 100644
--- a/src/plugin-interface/QvPluginBase.hpp
+++ b/src/plugin-interface/QvPluginBase.hpp
@@ -6,6 +6,10 @@
 #include <QVariant>
 #include <memory>
 
+#ifndef Q_DECL_ENUMERATOR_DEPRECATED_X
+#define Q_DECL_ENUMERATOR_DEPRECATED_X(x)
+#endif
+
 constexpr auto PLUMBUM_PLUGIN_INTERFACE_VERSION = 3;
 constexpr auto PLUMBUM_PLUGIN_INTERNAL_PROPERTY_KEY = "_PLUMBUM_PLUGIN_OPTIONS_";
